lib/controllers/BaseController.js

```diff
@@ -35,6 +35,12 @@ class BaseController {
     }
   }
 
+  validateDateString(isoDateString) {
```

Comment:
shouldn't this be a static method in utils ?

Reply:
it is called by constructor object of BaseController's child class.

Reply:
Hmm , I see there is another such method in BaseController for getuuid , I guess both should have ideally been in some util class as these are reusable methods, we can leave this for now.